app/components/site-chrome/template.hbs

```diff
@@ -124,7 +124,7 @@ Using double link-to in order to get an active class on the li tag
           </li>
 
           <li class="list-item list-item--nav">
-            <a href="/story/class-2021/" class="inherit gtm__click-tracking" data-action="Side Navigation">Artist Propulsion Lab</a>
+            <a href="https://www.wqxr.org/shows/artist-propulsion-lab-show/" target="_blank" rel="noopener noreferrer" class="inherit gtm__click-tracking" data-action="Side Navigation">Artist Propulsion Lab</a>
```

Comment:
do we need to have the "https://www.wqxr.org" in the href?

Reply:
no, I'm dumb